cob_phidgets/ros/src/phidgetik_ros.cpp

```diff
@@ -222,7 +224,7 @@ auto PhidgetIKROS::update() -> void
 		if(_indexNameMapItr != _indexNameMapDigitalIn.end())
 			name = (*_indexNameMapItr).second;
 		names.push_back(name);
-		states.push_back(this->getInputState(i));
+        states.push_back(!this->getInputState(i)); // Inverted by the board for unknown reasons
```

Comment:
If I directly connect input and output, the states are only equal if inversion takes place. However, if nothing is connected the input state is always high (pull-up)